bin/deploy

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ENV=$1
+BRANCH_NAME="${BRANCH:-main}"
+REPOSITORY="${REPO:-dpul-collections}"
+JOB_NAME="${JOBNAME:-dpulc}"
+if [[ -z ${ENV} ]];
+then
+  echo "Missing Environment. Command: BRANCH=main ./bin/deploy staging"
+  exit
+fi
+# Notify Github of Starting Deploy, Deploy, Notify Github of Ending Deploy
+ssh -qtt deploy@nomad-host-prod1.lib.princeton.edu << EOF
+  DEPLOY_ID=\$(curl -X POST -H "Accept: application/vnd.github+json-H" -H "Content-Type: application/x-www-form-urlencoded" -H "Authorization: Bearer \$GITHUB_TOKEN" --data "{\"ref\":\"${BRANCH_NAME}\",\"description\":\"Deploy from Nomad script\", \"auto_merge\": false, \"environment\": \"${ENV}\", \"required_contexts\": [] }" "https://api.github.com/repos/pulibrary/${REPOSITORY}/deployments" | jq .id)
+  curl -s -X POST -H "Accept: application/vnd.github+json-H" -H "Content-Type: application/x-www-form-urlencoded" -H "Authorization: Bearer \$GITHUB_TOKEN" --data "{\"environment\":\"${ENV}\",\"state\":\"in_progress\",\"log_url\": \"https://nomad.lib.princeton.edu/ui/jobs/${JOB_NAME}-${ENV}\", \"description\":\"Deployment finished successfully.\"}" "https://api.github.com/repos/pulibrary/${REPOSITORY}/deployments/\$DEPLOY_ID/statuses"
```

Comment:
typo "deployment finished successfully" should be "deployment started" or something.